### PR TITLE
Add gtk4 files

### PR DESCRIPTION
**Pull request by sfanxiang**
Squashed from https://github.com/sfanxiang/gtk-rs-sys/commits/gtk4, due to large merge conflicts.

- [ ] Appveyor CI is not available
- [x] Travis CI depends on having https://github.com/gtk-rs/examples4
- [x] Depends on https://github.com/gtk-rs/gir-files/pull/39

### CODE REVIEW

File: gsk4-sys/LICENSE [hunk 1]
```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
+
+Copyright (c) 2013-2015, The Gtk-rs Project Developers.
```
Comment (sdroege) [inline gsk4-sys/LICENSE]: The year is 2019 :)

**Comment by sdroege** [comment]
Looks good to me. For the Travis CI, why does it depend on the example? Can't we build things without the GTK4 examples for now and once everything is in place enable the examples?

**Comment by sfanxiang** [comment]
@sdroege Dropped gtk4 examples.

**Comment by alatiera** [comment]
Looks good to me as well.

**Comment by sdroege** [comment]
What has to be done for appveyor here? I see that it's green, but travis is not :)

**Comment by EPashkin** [comment]
Appveyor don't have regen check
IMHO this PR missed 3 thing: 
1. update rust to 1.36.0 in .travis.yml,
1. update submodules
1. full regen: old version.txt need changed too.

**Comment by EPashkin** [comment]
@sfanxiang Thanks

**Comment by EPashkin** [comment]
👍

**Comment by sdroege** [comment]
Let's get this in then, I'd say? :)

**Comment by sfanxiang** [comment]
Hi all, what's the status on this? Can we please merge this or is there anything I'm supposed to do?

Thanks for reviewing, by the way.

**Comment by sdroege** [comment]
> Hi all, what's the status on this? Can we please merge this or is there anything I'm supposed to do?

I don't know, it was ready from my side 6 days ago already.

@GuillaumeGomez ?

**Comment by GuillaumeGomez** [comment]
Arf sorry! Might have missed the notification (don't hesitate to ping us in such cases!)...

**Pull request merged**